tests/foreman/destructive/test_katello_certs_check.py

```diff
@@ -79,64 +77,6 @@ def test_positive_update_katello_certs(cert_setup_destructive_teardown):
         assert result.status == 0, 'Not all services are running'
 
 
-@pytest.mark.parametrize(
-    'certs_vm_setup',
-    [
-        {'nick': 'rhel7', 'target_memory': '20GiB', 'target_cores': 4},
-        {'nick': 'rhel8', 'target_memory': '20GiB', 'target_cores': 4},
-    ],
-    ids=['rhel7', 'rhel8'],
-    indirect=True,
-)
```

Comment:
Can you keep this parametrization for el7 and el8 for moved test?

Reply:
Nope. 
We have separate job for RHEL7 and RHEL8 :smile:

Reply:
Great catch!